interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -628,6 +634,47 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  //-----------------------------------------------------------------------------
+  // The tests below are used by gRPC interop test framework.
```

Comment:
This comment really doesn't tell me anything, as I would think this class is part of that framework. Maybe just remove the comment, and people can find the usages of the method if they wonder why it is here?